packages/common/utils/src/filter-characters.ts

```diff
@@ -0,0 +1,12 @@
+export function filterCharacters(text: string, validCharacters: string[] = []): string {
```

Comment:
I think this signature is very confusing and not very flexible, I will try to think about the proposal